gr-filter/lib/fft_filter.cc

```diff
@@ -100,8 +101,12 @@ void fft_filter_fff::compute_sizes(int ntaps)
     d_nsamples = d_fftsize - d_ntaps + 1;
 
     if (VERBOSE) {
-        std::cerr << "fft_filter_fff: ntaps = " << d_ntaps << " fftsize = " << d_fftsize
+        gr::logger_ptr logger, debug_logger;
+        gr::configure_default_loggers(logger, debug_logger, "fft_filter_fff");
```

Comment:
Um, setting up a logger in every call to `compute_sizes` seems like a bad idea.

Do you perhaps do this because the logger needs to be set up before this, and you forgot to do this in the constructor instead?

Reply:
addressed in #3024